src/plugins/discover/server/ui_settings.ts

```diff
@@ -165,6 +166,18 @@ export const uiSettings: Record<string, UiSettingsParams> = {
     category: ['discover'],
     schema: schema.arrayOf(schema.string()),
   },
+  [DOC_TABLE_LEGACY_TABLE]: {
+    name: i18n.translate('discover.advancedSettings.docTableVersionName', {
+      defaultMessage: 'Use legacy table',
+    }),
+    value: true,
+    description: i18n.translate('discover.advancedSettings.docTableVersionDescription', {
+      defaultMessage:
+        'Prefer the legacy version of the documents table in Discover while available',
```

Comment:
I don't get this message well. Do we want to prompt the users to use the legacy table for a while?

Reply:
this is a more kind of historic message, from back in the days when it started, we should adapt this

dear @gchaps, when this is merged, the user can chose a different document table in Discover, which will be the default later on, and for the first time, users should be able to switch to the legacy table. What's the best way to message this? Many thx!